### PR TITLE
Frontend-Assesment-1

### PR DESCRIPTION
**Pull request by Neerp25**

### CODE REVIEW

**Comment by Neerp25** [comment]
image
image
image

File: frontend/assessments/1/style.scss [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    font-family: "Poppins", sans-serif;
+    font-weight: 500;
+    font-style: italic;
+}
+
+.Header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
```
Comment (rishav-kd) [inline frontend/assessments/1/style.scss:21]: Use rem/em for styling and put common ones in a separate variables file.

File: frontend/assessments/1/style.scss [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    font-size: 24px;
+}
+
+.dropdown {
+    display: none;
+    flex-direction: column;
+    position: absolute;
+    top: 60px;
+    left: 0;
+    background-color: rgb(233, 140, 155);
```
Comment (rishav-kd) [inline frontend/assessments/1/style.scss:52]: use variables

**Review by rishav-kd — COMMENTED** [review]
marks deducted for - i) missing hamburger for mobile (-15) ii) navbar should be sticky (-10) iii) missing variable.scss files (-5) iv) use nested styling (-5) v) minor styling issues (-5) vi) missing drift video (-10)

**Pull request closed without merging**